content/60.methods.md

```diff
@@ -1,11 +1,44 @@
 ## Methods
 
+In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becomes available, this project is an open, collaborative effort that invited contributions the scientific community broadly, similar to previous efforts to develop collaborative reviews [@doi:10.1098/rsif.2017.0387; @url:https://greenelab.github.io/deep-review].
+Contributors were recruited by word of mouth and on Twitter.
+Several efforts to work with existing efforts to train early-career scientists were also integrated: Appendix A contains summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine [@url:https://github.com/ismms-himc/covid-19_sinai_reviews] and two of the authors were recruited through the American Physician Scientist Association's Virtual Summer Research Program [@url:https://www.physicianscientists.org/page/summer-research-pilot-program].
+The project was managed through GitHub [@url:https://github.com/greenelab/covid19-review] using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate a version of the manuscript online [@url:https://greenelab.github.io/covid19-review].
+Contributors developed text that was proposed through GitHub's pull request system and then reviewed and approved by at least one other author.
+While this document reflects the current version of record, the online version will continue to be developed as information about the pandemic emerges.
+Below, we will describe the processes used to synthesize the literature.
+
+### Technical Infrastructure
+
+#### Collaborative Writing and Manuscript Generation
+
+Manubot [@doi:10.1371/journal.pcbi.1007128] is a collaborative writing framework developed to adapt open-source software development techniques and version control for manuscript development.
+Here, Manubot was used to generate a manuscript from text maintained using GitHub, a popular, online version control interface.
+The GitHub implementation allowed users to contribute either using git on the command line or using the GitHub user interface.
+Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs [@doi:10.1371/journal.pcbi.1007128].
+Due to the needs of this project, project contributors also submitted pull requests to add support for clinical trial identifiers.
+<!-- Additional info-->
+Finally, the use of Manubot and GitHub allowed for scripted updates to be run each time the manuscript was generated.
+These updates were used to check that the manuscript was complete and to dynamically update information in the manuscript.
+<!-- Additional info about CI-->
+
+#### Data Analysis and Visualization
+
+The combination of Manubot and GitHub Actions made it possible to integrate up-to-date analyses and visualizations of online data sources into the manuscript.
```

Comment:
Maybe we could get a brief blurb from @agitter in a subsequent PR describing the branching strategy. I thought that was a very clever solution that used the github platform for hosting.

Reply:
See #621